command/ca/renew.go

```diff
@@ -0,0 +1,415 @@
+package ca
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/smallstep/certificates/api"
+	"github.com/smallstep/certificates/ca"
+	"github.com/smallstep/cli/command"
+	"github.com/smallstep/cli/crypto/pemutil"
+	"github.com/smallstep/cli/crypto/pki"
+	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/flags"
+	"github.com/smallstep/cli/ui"
+	"github.com/smallstep/cli/utils"
+	"github.com/urfave/cli"
+)
+
+func renewCertificateCommand() cli.Command {
+	return cli.Command{
+		Name:   "renew",
+		Action: command.ActionFunc(renewCertificateAction),
+		Usage:  "renew a valid certificate",
+		UsageText: `**step ca renew** <crt-file> <key-file>
+		[**--ca-url**=<uri>] [**--root**=<file>]
+		[**--out**=<file>] [**--expires-in**=<duration>] [**--force**]`,
+		Description: `
+**step ca renew** command renews the given certificates on the certificate
+authority and writes the new certificate to disk either overwriting <crt-file>
+or using a new file if the **--out**=<file> flag is used.
+å
```

Comment:
weird character. I'll do a grammatical once over once I'm through reading the rest of the commit.

Reply:
Fixed.